spec/integration/puck_spec.rb

```diff
@@ -62,8 +73,15 @@ def jar_command(cmd)
   end
 
   it 'exposes JRuby\'s bin files' do
-    output = isolated_run(jar_command('irb -h'))
-    output.should include('Usage:  irb.rb')
+    output = isolated_run(jar_command('gem -h'))
+    output.should include('gem -h/--help')
+  end
+
+  it "can display IRB help message (#{RewindInJars.behavior_after_gets})" do
+    pending 'rewindable files inside jar files', if: RewindInJars.behavior_after_gets == :discard do
```

Comment:
The spec description should be changed too, just having a parentheses with "discard" or "ignore" in it is confusing, change it to something like "runs bin files that require embedded files to be rewindable (current behaviour: discard)" (followed by something like "PENDING: not supported")